src/core/utils/constants.ts

```diff
@@ -8,3 +8,5 @@ export const WORKSPACE_TYPE = 'workspace';
 export const WORKSPACE_PATH_PREFIX = '/w';
 
 export const PUBLIC_WORKSPACE_ID = 'public';
+
+export const DEFAULT_WORKSPACE_ID = 'default';
```

Comment:
In 2.13 I thought `default workspace` is a virtual workspace? Could we add some comment on it as at first I thought it is a new physical workspace.

Reply:
sounds good, comments added.